bot/features/music/music.py

```diff
@@ -297,6 +304,45 @@ async def resume(self, ctx):
         if self.voice_client.is_paused():
             self.voice_client.resume()
 
```

Comment:
I like what you are doing here, however I think we can actually handle this with subcommands.

[Check the utility cog here](https://github.com/The-Galaxy-Aces/Discord-Bot/blob/main/bot/features/utility/utility.py#L73-L82)

What I think we can do is set music repeat as a group like utility is setup, then each argument passed to music repeat will be its own subcommand, so there would be two in total with several aliases for each.

I think something like this could work.
```suggestion    
    @music.group(aliases=['rp'])
    @TGACog.check_permissions()
    async def repeat(self, ctx):
        '''
        Assorted music repeat commands
        '''

    @repeat.command(aliases=['everything', 'queue'])
    @TGACog.check_permissions()
    async def all(self, ctx, *args):
        # Repeat all code
        
    @repeat.command(aliases=['this', 'current'])
    @TGACog.check_permissions()
    aysnc def song(self, ctx, *args):
        # Repeat song code
```